Biblio.bib

```diff
@@ -1,3 +1,26 @@
+@Misc{EIP-100,
+	url = "https://github.com/ethereum/EIPs/blob/f3a591f6718035ba358d6a479cadabe313f6ed36/EIPS/eip-100.md",
+	author = "Vitalik Buterin",
+	title = "EIP-100: Change difficulty adjustment to target mean block time including uncles",
+	year = "2016",
+	month = "April",
+}
+
+@Misc{EIP-649,
+	url = "https://github.com/ethereum/EIPs/blob/63c93eb71d1148435fa2cadb36da7df2bd865a3d/EIPS/eip-649.md",
+	author = "Afri Schoedon and Vitalik Buterin",
+	title = "EIP-649: Metropolis Difficulty Bomb Delay and Block Reward Reduction",
+	year = "2017",
+	month = "June",
+}
+
+@Misc{EIP-2,
+	url = "https://github.com/ethereum/EIPs/blob/master/EIPS/eip-2.md",
```

Comment:
Why do other URLs contain `/blob/<hash>` while this one has `/blob/master`?

Reply:
Changed. It did cross my mind at the time but I didn't think more about it.